dlp/api/Snippets/ClientLibraryQuickStart.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_quickstart]
+
+using System;
+using System.Collections.Generic;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+
+public class ClientLibraryQuickStart
+{
+    public static InspectContentResponse QuickStart(
+        string projectId,
+        string text,
+        IEnumerable<InfoType> infoTypes = null)
+    {
+        // Instantiate a client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the byte content item by specifying the type of data to be inspected
+        // and data to be inspected.
+        var byteContentItem = new ByteContentItem
+        {
+            Type = ByteContentItem.Types.BytesType.TextUtf8,
+            Data = Google.Protobuf.ByteString.CopyFromUtf8(text)
+        };
+
+        // Construct content item by setting byte item.
+        var contentItem = new ContentItem { ByteItem = byteContentItem };
+
+        // Set minimum likelihood and limits.
+        var minLikelihood = Likelihood.Possible;
+        var limits = new InspectConfig.Types.FindingLimits
+        {
+            MaxFindingsPerRequest = 0
```

Comment:
good to mention that 0 means use the maximum allowed

Reply:
Done. Thanks.

Reply:
important to mention maximum **allowed**, because the system has a limit of 3,000 findings
https://cloud.google.com/dlp/docs/reference/rest/v2/InspectConfig#findinglimits